truetype/truetype_test.go

```diff
@@ -110,7 +110,18 @@ func TestParse(t *testing.T) {
 
 func TestIndex(t *testing.T) {
 	testCases := map[string]map[rune]Index{
-		"luxisr": {
+		"luxisr.ttf": {
+			' ':      3,
+			'!':      4,
+			'A':      36,
+			'V':      57,
+			'É':      101,
+			'ﬂ':      193,
+			'\u22c5': 385,
+			'中':      0,
+		},
+
+		"x-NotoSansCJK-Black.ttc": {
```

Comment:
If you're going to add an "x-foo" font, you need to update the testdata/make-other-hinting-txts.sh script.